sale_import_rest/tests/test_sale_import_rest.py

```diff
@@ -0,0 +1,38 @@
+#  Copyright (c) Akretion 2020
+#  License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
+
+import requests
+
+import odoo.tools
+from odoo.tests import HttpCase
+
+from odoo.addons.base_rest.tests.common import BaseRestCase
+from odoo.addons.sale_import_base.tests.common_sale_order_import import SaleImportCase
+
+
+@odoo.tests.common.at_install(False)
+@odoo.tests.common.post_install(True)
+class TestSaleOrderImport(SaleImportCase, HttpCase, BaseRestCase):
+    def setUp(self):
+        super().setUp()
+        self.setUpRegistry()
+        host = "127.0.0.1"
+        port = odoo.tools.config["http_port"]
+        self.url = "http://%s:%d/sale-import/import" % (host, port)
+        # self.request_content = "{
```

Comment:
This is for testing API calls. Does this make sense ? If yes, is there an example somewhere ? In Shopinvader there are examples with GET but not with POST, plus I am having other issues so I'm not sure. 

Or should I use the component/service directly ?